util/postman/openkit.postman_environment.json

```diff
@@ -0,0 +1,123 @@
+{
+	"id": "b4c4d4a9-7480-4823-98a2-60bdf5b7f4fe",
+	"name": "openkit - prod",
```

Comment:
nit: Is this to distinguish between partner's dev and prod environments (like for Landing)? Could we just leave it as `openkit`?

Reply:
Agreed, will keep it as `openkit` for now. In the future we will support multiple environments (hence the URLs will be different), but agree that for now we can simplify it.